http-common/src/server.rs

```diff
@@ -344,24 +348,23 @@ mod fake_server {
 		}
 	}
 
-	use crate as http_common;
 
 	http_common::make_server! {
 		server: Server,
 		api_version: ApiVersion,
 		routes: [
-			fake_route::Route,
+			test_route::Route,
 		],
 	}
 
-	pub(crate) struct Server;
+	pub struct Server;
```

Comment:
Err, what I meant is, can it be just `struct Server;`, ie neither `pub(crate)` nor `pub`?

Same for `ApiVersion`

Reply:
oopsie, yep, true.